### PR TITLE
Linting solidity and javascript

### PR DESCRIPTION
**Pull request by dimpar**
In this PR we are updating linting for Solidity contracts and JS files to stay consistent across keep-network repositories.

### CODE REVIEW

File: solidity/package.json [hunk 1]
```diff
@@ -21,8 +21,8 @@
     "lint:fix": "npm run lint:fix:js && npm run lint:fix:sol",
     "lint:js": "eslint ${npm_package_config_eslintPaths}",
     "lint:fix:js": "eslint --fix ${npm_package_config_eslintPaths}",
-    "lint:sol": "solium -d contracts/ && prettier --list-different **/*.sol",
-    "lint:fix:sol": "solium -d contracts/ --fix && prettier --write **/*.sol"
+    "lint:sol": "solium -d contracts/ && prettier --list-different contracts/ **/*.sol",
+    "lint:fix:sol": "solium -d contracts/ --fix && prettier --write contracts/ **/*.sol"
```
Comment (nkuba) [inline solidity/package.json:25]:
```suggestion
    "lint:sol": "solium -d contracts/ && prettier --check '**/*.sol'",
    "lint:fix:sol": "solium -d contracts/ --fix && prettier --write '**/*.sol'"
```
See:
https://github.com/keep-network/keep-core/pull/2244#discussion_r556629670
https://github.com/keep-network/keep-core/pull/2244#discussion_r556625475

File: solidity/package.json [hunk 2]
```diff
@@ -21,8 +21,8 @@
     "lint:fix": "npm run lint:fix:js && npm run lint:fix:sol",
     "lint:js": "eslint ${npm_package_config_eslintPaths}",
     "lint:fix:js": "eslint --fix ${npm_package_config_eslintPaths}",
```
Comment (nkuba) [inline solidity/package.json:23]: Would removing `eslintPaths` configuration work here?
```suggestion
    "lint:js": "eslint .",
    "lint:fix:js": "eslint --fix .",
```
See https://github.com/keep-network/keep-core/pull/2244#discussion_r556652567

File: staker-rewards/package.json [hunk 3]
```diff
@@ -11,8 +11,8 @@
     "test": "NODE_ENV=test node --experimental-json-modules node_modules/mocha/bin/mocha",
     "lint": "npm run lint:js",
     "lint:fix": "npm run lint:fix:js",
-    "lint:js": "eslint .",
-    "lint:fix:js": "eslint --fix ."
+    "lint:js": "eslint ${npm_package_config_eslintPaths}",
+    "lint:fix:js": "eslint --fix ${npm_package_config_eslintPaths}"
```
Comment (nkuba) [inline staker-rewards/package.json:15]: No, we don't need this change, dosn't `eslint .` work?

**Review by nkuba — CHANGES_REQUESTED** [review]

File: solidity/package.json (hunk 1 above)
Comment (dimpar) [reply solidity/package.json]: Done a13b562

File: solidity/package.json (hunk 2 above)
Comment (dimpar) [reply solidity/package.json]: Done a3ab8b7

File: staker-rewards/package.json (hunk 3 above)
Comment (dimpar) [reply staker-rewards/package.json:15]: Yup, it works, done in a3ab8b7

File: .pre-commit-config.yaml [hunk 4]
```diff
@@ -6,18 +6,18 @@ repos:
...
     - id: go-lint
  - repo: https://github.com/keep-network/pre-commit-hooks.git
-   rev: 63e729f
+   rev: v1.3.0
    hooks:
     - id: check-added-large-files
  - repo: local
    hooks:
-    - id: solidity-lint-js
+    - id: lint-js
```
Comment (nkuba) [inline .pre-commit-config.yaml:14]: Let's leave the `solidity-` prefix as we're still in the solidity directory.

File: .pre-commit-config.yaml [hunk 5]
```diff
@@ -6,18 +6,18 @@ repos:
...
       name: 'solidity lint js'
       entry: /usr/bin/env bash -c "cd solidity && npm run lint:js"
-      files: '\.js$'
+      files: 'solidity\/.*\.js$'
       language: script
       description: "Checks JS code according to the package's linter configuration"
-    - id: solidity-lint-sol
+    - id: lint-sol
```
Comment (nkuba) [inline .pre-commit-config.yaml:20]: Let's leave the `solidity-` prefix as we're still in the solidity directory.

File: .pre-commit-config.yaml [hunk 6]
```diff
@@ -26,6 +26,5 @@ repos:
     - id: staker-rewards-lint-js
       name: 'staker rewards lint js'
       entry: /usr/bin/env bash -c "cd staker-rewards && npm run lint:js"
-      files: '\.js$'
```
Comment (nkuba) [inline .pre-commit-config.yaml:29]: We should leave the `files` filter here and maybe add `staker-rewards` directory to the pattern?

File: staker-rewards/package.json (hunk 3 above)
Comment (nkuba) [reply staker-rewards/package.json:15]: a3ab8b7 updates `solidity/package.json`, we still need to update `staker-rewards/package.json`.

**Review by nkuba — CHANGES_REQUESTED** [review]

File: .pre-commit-config.yaml (hunk 4 above)
Comment (dimpar) [reply .pre-commit-config.yaml]: Done in 46324f9

File: .pre-commit-config.yaml (hunk 5 above)
Comment (dimpar) [reply .pre-commit-config.yaml]: Done

File: .pre-commit-config.yaml (hunk 6 above)
Comment (dimpar) [reply .pre-commit-config.yaml:29]: Sure, done in 46324f9

File: staker-rewards/package.json (hunk 3 above)
Comment (dimpar) [reply staker-rewards/package.json]: a5f1e04

**Review by nkuba — APPROVED** [review]

**Pull request merged**